app/scripts/app.js

```diff
@@ -552,6 +552,9 @@ function udbAppConfig(
         'titleSuffix': ' | Gebruiker bewerken'
       }
     })
+    .state("split.organizerOwnership", {
+      url: "/organizer/:id/ownership",
```

Comment:
In the ticket the url is with an "s" but maybe you discussed this.
```suggestion
      url: "/organizer/:id/ownerships",
```